tests/test_github.py

```diff
@@ -227,9 +227,7 @@ def test_github_get_token_from_file_in_home_directory(self, mocker: MockerFixtur
         github_token_contents = ('ghp_mytoken_for_github_com\n'
                                  'ghp_myothertoken_for_git_example_org git.example.org\n'
                                  'ghp_yetanothertoken_for_git_example_com git.example.com')
-        _mock_open = mock_open(read_data=github_token_contents)
-        _mock_open.return_value.readlines.return_value = github_token_contents.split('\n')
```

Comment:
...but the test didn't capture that, coz `split('\n')` removes `\n`, unlike `readlines()`.

In fact, this extra mocking of `readlines` was unnecessary, `mock_open` already fills up methods like `readlines`.